frontend/tests/features/collection/collection.test.ts

```diff
@@ -20,29 +19,6 @@ const TEST_COLLECTION = {
 };
 
 describe("Collection", () => {
-  describe("Collection Creation Modal", () => {
```

Comment:
Thanks for deleting these unit tests in the smoke test! IIRC the DOI validation is now done in BE?

Thank you!